src/pke/include/scheme/cryptocontextparams-base.h

```diff
@@ -94,6 +94,13 @@ class Params {
 
     // max batch size of messages to be packed in encoding
     usint batchSize;
+    
+    //PRE security mode
+    ProxyReEncryptionMode PREMode;
+    
+    // cyclotomic order. For power of two case
+    // cyclotomic order = 2 * ring dimension
+    usint cyclOrder;
```

Comment:
cyclOrder has been removed, do not restore it

Reply:
I think this came from the rebase from a previous commit, I've removed it.